docs/sql/ddl.txt

```diff
@@ -829,10 +833,14 @@ already existing user-created doc tables in the cluster::
 
     cr> show create table my_table;
     +-----------------------------------------------------+
-    | SHOW CREATE TABLE my_table                          |
+    | SHOW CREATE TABLE doc.my_table                      |
```

Comment:
the fqn of a table now contains the "doc" schema prefix when using default schema: https://github.com/crate/crate/commit/d07e16b9ab0041f786b90d38dbc7d55834bc0cb3#diff-75c7a5c63d9bccb8bfda833ed7d4c564L84